packages/aws-rfdk/lib/deadline/test/render-queue.test.ts

```diff
@@ -1904,6 +2014,42 @@ describe('RenderQueue', () => {
       });
     });
 
+    test('not specified with TLS', () => {
+      // GIVEN
+      const isolatedStack = new Stack(app, 'IsolatedStack');
+
+      const props: RenderQueueProps = {
+        images,
+        repository,
+        version: new VersionQuery(isolatedStack, 'Version'),
+        vpc,
+        trafficEncryption: {
+          externalTLS: {
+          },
+        },
+      };
+
+      const renderQueue = new RenderQueue(isolatedStack, 'RenderQueue', props);
+
+      const loadBalancerLogicalId = dependencyStack.getLogicalId(
+        renderQueue.loadBalancer.node.defaultChild as CfnElement,
+      );
+
+      expectCDK(isolatedStack).to(haveResource('AWS::Route53::RecordSet', {
+        // eslint-disable-next-line dot-notation
+        Name: `${RenderQueue['DEFAULT_HOSTNAME']}.${RenderQueue['DEFAULT_DOMAIN_NAME']}.`,
```

Comment:
I think this is a scenario where we would duplicate the values of the expected FQDN here rather than referencing the internals of the `RenderQueue` class. This makes the tests anchor to the expected DNS record and protects someone from changing the default and breaking things for users.

Reply:
Makes sense